RussianFilter/sections/specific.txt

```diff
@@ -26,6 +26,7 @@ sport-express.ru##section[class$="mainpage__column-left"] > div[class$="news-fee
 infobank.by##.homePageBanner
 newstudio.tv##span[id^="pp_"] > a[target="_blank"] > var > img[alt]
 newstudio.tv##a[href^="http://cutter.li/"]
+pornosexrolik.info##.fc-right
```

Comment:
Looks more like an Annoyance than an Ad to me.

Reply:
On my side this element doesn't load correctly. I moved that to annoyances.